src/sdk/bucketspace_fs.js

```diff
@@ -440,15 +440,56 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     ////////////////////
 
     async put_bucket_tagging(params) {
-        // TODO
+        try {
+            const { name, tagging } = params;
+            dbg.log0('BucketSpaceFS.put_bucket_tagging: Bucket name, tagging', name, tagging);
+            const bucket_config_path = this._get_bucket_config_path(name);
+            const { data } = await nb_native().fs.readFile(this.fs_context, bucket_config_path);
+            const bucket = JSON.parse(data.toString());
+            bucket.tag = tagging;
+            nsfs_schema_utils.validate_bucket_schema(bucket);
+            await update_config_file(
+                this.fs_context,
+                this.bucket_schema_dir,
+                bucket_config_path,
+                JSON.stringify(bucket),
+            );
+        } catch (error) {
+            throw this._translate_bucket_error_codes(error);
+        }
     }
 
     async delete_bucket_tagging(params) {
-        // TODO
+        try {
+            const { name } = params;
+            dbg.log0('BucketSpaceFS.delete_bucket_tagging: Bucket name', name);
+            const bucket_config_path = this._get_bucket_config_path(name);
+            const { data } = await nb_native().fs.readFile(this.fs_context, bucket_config_path);
+            const bucket = JSON.parse(data.toString());
+            delete bucket.tag;
+            nsfs_schema_utils.validate_bucket_schema(bucket);
+            await update_config_file(
+                this.fs_context,
+                this.bucket_schema_dir,
+                bucket_config_path,
+                JSON.stringify(bucket),
+            );
+        } catch (error) {
+            throw this._translate_bucket_error_codes(error);
+        }
     }
 
     async get_bucket_tagging(params) {
-        // TODO
+        try {
+            const { name } = params;
+            dbg.log0('BucketSpaceFS.get_bucket_tagging: Bucket name', name);
+            const bucket_config_path = this._get_bucket_config_path(name);
+            const { data } = await nb_native().fs.readFile(this.fs_context, bucket_config_path);
+            const bucket = JSON.parse(data.toString());
```

Comment:
would be nice to move this common code that repeats throughout this file to a method like `const bucket = await this._load_bucket(name);`. maybe in a separate pr? CC @romayalon

Reply:
it was fixes as part of the configFS changes, thanks